Composer/packages/adaptive-form/src/components/fields/StringField.tsx

```diff
@@ -89,7 +95,7 @@ export const StringField: React.FC<FieldProps<string>> = function StringField(pr
         placeholder={placeholder}
         readOnly={readonly}
         styles={{
-          ...borderStyles(Boolean(transparentBorder), Boolean(error)),
+          ...borderStyles(Boolean(transparentBorder), Boolean(error), hasIcon),
```

Comment:
!!

Reply:
??

Reply:
!!hasIcon

Reply:
I don't know what difference that makes but ok